tests/integration/it_reservations_test.py

```diff
@@ -7,35 +7,29 @@
 from timeless.restaurants.models import Reservation
 
 
-@pytest.mark.skip("reservations.list does not implemented yet")
 def test_list(client):
     reservation = factories.ReservationFactory()
     response = client.get(url_for("reservations.list"))
     assert response.status_code == HTTPStatus.OK
-    assert reservation.comment in response.data
+    assert reservation.comment in response.data.decode('utf-8')
```

Comment:
@vladarefiev can you also assert that the response contains only 1 reservation and not more?

Reply:
@amihaiemil ok